src/main/java/org/sagebionetworks/bridge/services/EnrollmentService.java

```diff
@@ -96,12 +97,17 @@ public Enrollment enroll(Enrollment enrollment) {
 
         // verify this has appId and accountId
         Validate.entityThrowingException(INSTANCE, enrollment);
+        
+        // Verify that the caller has access to this study
+        AuthUtils.checkStudyScopedToCaller(enrollment.getStudyId());
```

Comment:
Unit test for caller not in study?

Reply:
This is tested by `enrollNotAuthorizedAsStudyResearcher()`